libpod/image/image.go

```diff
@@ -37,6 +37,8 @@ type imageConversions struct {
 	storeRef types.ImageReference
 }
 
+const ImageDigestType = "sha256"
```

Comment:
Lint is going to complain this doesn't have a godoc comment

Reply:
yeah, pulled trigger too quick